pybravia/client.py

```diff
@@ -436,6 +449,15 @@ async def get_playing_info(self) -> dict[str, Any]:
         result = resp.get("result", [{}])[0]
         return result
 
+    async def set_picture_setting(self, target: str, value: str | None = None) -> bool:
```

Comment:
`value` must only be a `string`.
```suggestion
    async def set_picture_setting(self, target: str, value: str) -> bool:
```